Prog/Housing forecast/PUMA TAZ Crosswalk and Mapping.Rmd

```diff
@@ -0,0 +1,390 @@
+---
+title: "PUMA-TAZ-Mapping"
+output: html_document
+date: "2024-05-09"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## R Markdown
+
+```{r libraries}
+library(tidyverse)
+library (dplyr)
+
+#install.packages("Rcpp")
+#library(Rcpp)
+#install.packages("tigris")
+library(tigris)
+
+#install.packages("tidycensus")
+library(tidycensus)
+
+
+#install.packages("sf")
+library(sf)
+
+library(ggplot2)
+#install.packages("remotes")
+#remotes::install_github("UrbanInstitute/urbnmapr", build_vignettes = TRUE)
+library(urbnmapr)
+library(urbnthemes)
+#remotes::install_github("r-spatial/mapview")
+library(mapview)
+library(units)
+library(leaflet)
+library(stringr)
+library(knitr)
+library(tidyr)
+library(janitor)
+
+# Only need to set once per script
+options(tigris_class = "sf")
+set_urbn_defaults(style = "map")
+
+
+```
+
+
+```{r read in data}
+
+## READ IN DATA
+
+dc_blocks_20 = 
+  blocks(
+    state= "District of Columbia", 
+    year = 2020) %>%
+  # rename the blocks population variables
+  mutate(BLOCK_POP20 = POP20, 
+         BLOCK_HOUSING20 = HOUSING20)
+
+dc_tracts_20 = 
+  tracts(
+    state= "District of Columbia", 
+    year = 2020)
+
+dc_pumas_12 = 
+  pumas(state = "District of Columbia", 
+        cb = FALSE, 
+        year = 2012)
+
+dc_pumas_23 = st_read(
+  dsn = "C:/DCData/Libraries/DMPED/Prog/Housing forecast/DC_PUMAs_2023/tl_2023_11_puma20.shp",
+  quiet = TRUE
+)
+
+  # Pulling in TAZ with pop data
+
+ # unzip(zipfile = "C:/Users/akort/Box/Greater DC/Projects/DMPED Housing Assessment 2024/Task 1 - Households Projections and Housing Demand/Projection Methodology/DC_Forecast_10_0_TAZ.zip", exdir = "C:/DCData/Libraries/DMPED/")
+  
+  dc_taz = st_read(
+      dsn = "C:/DCData/Libraries/DMPED/Prog/Housing forecast/DC Forecast TAZ/DC_Forecast_10_0_TAZ.shp", 
```

Comment:
you should be pulling this from "\\sas1\dcdata\Libraries\Planning\Raw\Forecasts\DC_Forecast_10_0_TAZ.shp"

Reply:
@akort15 this still needs to be changed to the Planning raw folder